apps/ledger-live-mobile/e2e/specs/speculos/addAccount/AddAccountADA.spec.ts

```diff
@@ -0,0 +1,4 @@
+import { runAddAccountTest } from "./AddAccount";
+import { Currency } from "@ledgerhq/live-common/e2e/enum/Currency";
+
+runAddAccountTest(Currency.ADA, "B2CQA-2500, B2CQA-2650, B2CQA-2678");
```

Comment:
do we need separate spec file for each currency? wouldn't it makes sense to do it in one file?

Reply:
Unfortunately, it's mandatory as Jest and Detox don't [parallelise](https://stackoverflow.com/questions/39831961/are-tests-inside-one-file-run-in-parallel-in-jest) tests in same files

Reply:
is it a problem though? we have the same issue with api tests where we use Jest and we don't mind it, the impact is not critical

Reply:
Testing time for all currencies in parallel in currently around 15min

Reply:
hm, ok let's have a ticket to investigate how we can improve parallelisation